Bump to v1.4.0

netoax opened the pull request:
What does this implement/fix? Explain your changes.
---------------------------------------------------
This PR bumps the current version to v1.4.0. The main changes are:

- Update the stack `yaml` files with the newer version of the KNoT services.
- Fix the `traeffik` version to be v1.7 since the latest (v2.x) isn't compatible with the current configuration.

Does this close any currently open issues?
------------------------------------------
Nops.

Any relevant logs, error output, etc?
-------------------------------------
Nops.

Any other comments?
-------------------
Nops.

ramonhpr commented on stacks/dev/stage-1.yml:
```diff
@@ -68,7 +68,7 @@ services:
   # Proxy
   traefik:
-    image: traefik
+    image: traefik:v1.7
```
This change doesn't seem to be related to the commit bump. 
Why do you need to fix the traefik version?

netoax replied:
I've created another commit related to this change: _51dd37a1267d517e13f04169ce7d6c7492cd3ccd_ .

ramonhpr commented on CHANGELOG.md:
```diff
@@ -1,3 +1,23 @@
+
+### Bug Fixes
+
+- Update packages to fix vulnerability issues
```
Update the traefik bug fix

netoax replied:
Done.

Commit:
Fix traefik version in v1.7

The newer traefik version (v2.x) changed most of the configuration
parameters. While the changes impact isn't assessed, it'd be better
and safier to fix the version with one that works fine in order to
avoid unexpected problems.

Commit:
Bump to v1.4.0
